low-level/files/opt/seagate/sspl/setup/sspl_reset.py

```diff
@@ -0,0 +1,157 @@
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import os
+import shutil
+import subprocess
+
+from cortx.sspl.lowlevel.framework.base.sspl_constants import (PRODUCT_FAMILY,
+ file_store_config_path, enabled_products)
+from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_setup import Cmd as SSPLSetup
```

Comment:
sspl_setup is the base program to call sspl_reset script.
In this case, when we import this sspl_reset in sspl_setup, this will become circular dependency. So find a way to avoid this.